std_msgs/Header header  # 正确：完整的消息类型
int32 sender_id
int32 q_level
int32 tile_size
int32 original_size
uint8[] data